include/EASTL/unique_ptr.h

```diff
@@ -336,9 +336,9 @@ namespace eastl
 		eastl::compressed_pair<pointer, deleter_type> mPair;
 
 		/// These functions are private in order to prevent copying, for safety.
-		unique_ptr(const this_type&);
-		unique_ptr& operator=(const this_type&);
-		unique_ptr& operator=(pointer pValue);
+		unique_ptr(const this_type&) = delete;
```

Comment:
Could you include this in a separate PR?  I don't feel like its related to the larger change.